.github/workflows/runTests.yml

```diff
@@ -0,0 +1,64 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  tests:
+    name: PHP ${{ matrix.php-versions }} on ${{ matrix.os }} w/ ${{ matrix.dependencies }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: ['7.3', '7.4']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        dependencies: ['install', 'update --prefer-lowest', 'update']
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up PHP ${{ matrix.php-versions }}
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+        extensions: mbstring
+        coverage: xdebug
```

Comment:
using pcov will speed up this build significantly

Reply:
Thanks for the reminder to test pcov out. Switched to it now.